### ISSUE TITLE
Sync review logs to Redash

### ISSUE THREAD

**Issue by rohangrover** [MoveOnOrg/eventroller#62]
Let's expose review logs on Redash for easier moderation and alerting. At a minimum, we want to sync reviewer logs for both messages to members and messages to hosts including:

- [ ] Sender/author/reviewer email address (or AKID or name or more than 1 of the above?)
- [ ] Timestamp
- [ ] Message text
- [ ] Recipient AKID's
- [ ] Status
- [ ] Log type (message vs. bulk message vs. note vs. bulk note)
- [ ] Visibility level

Since notes use the same object, let's add notes in there too. If you find any other data that could be useful and would be straightforward to add, please suggest it here.

**Comment by azuzunaga** [comment]
Post-deploy, update flydata to sync the following tables:
* `reviewer_reviewlog`
* `reviewer_review`
* `actionkit_membersearch_tag`

**Comment by rohangrover** [comment]
Does this include adding/removing tags @azuzunaga?

**Comment by azuzunaga** [comment]
Yes, @rohangrover. We can see that information in the `reviewer_review` table, columns `created_at` and `obsoleted_at`.

**Comment by rohangrover** [comment]
Sweet!
